### PR TITLE
User schema(db)

### PR DESCRIPTION
**Pull request by MikeSewell**
to run this locally, you need to delete the tables in your MySQL DB manually before you `docker-compose up --build`

### CODE REVIEW

File: db/migrations/20180306015519-create-user.js [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
       email: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
       },
-      uuid: {
-        type: Sequelize.STRING
+      points: {
+        type: Sequelize.INTEGER,
+      },
+      messages: {
```
Comment (drubiodev) [inline db/migrations/20180306015519-create-user.js]: suggestion how about `messagesCount`

Comment (MikeSewell) [reply db/migrations/20180306015519-create-user.js]: ok ill make those changes

**Comment by MikeSewell** [comment]
[link to why I used two files](https://github.com/sequelize/cli/issues/133)

screen shot 2018-03-07 at 2 44 14 pm

File: bot/client.js [hunk 2]
```diff
@@ -57,4 +57,10 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
+  member.sendMessage('Welcome to the channel!');
```
Comment (reactivepixel) [inline bot/client.js]: Why is this feature in your DB scheme branch? This was never merged into the dev branch, so it should not be here unless you have not followed the DWA branch model.

File: db/migrations/add-column-memberCount.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.addColumn("Members", "messagesCount", {
+      type: Sequelize.INTEGER,
```
Comment (reactivepixel) [inline db/migrations/add-column-memberCount.js:5]: Does this column matter as per my previous comments about not needing to count the messages, but rather apply a point transformation onto the user's point total upon a message being sent

Comment (MikeSewell) [reply db/migrations/add-column-memberCount.js:5]: We wanted to track who has sent the most messages as well. Of course, this won't start tracking until the merge. But as a reward, we want to track message count.

**Pull request closed without merging**